.github/workflows/scripts/linkerd_deploy.sh

```diff
@@ -16,10 +16,10 @@ fi
 
 curl -fsL https://run.linkerd.io/emojivoto.yml --output emojivoto.yml
 sleep 10
-mesheryctl system login --provider None
-mesheryctl mesh deploy adapter meshery-linkerd:10001
+#mesheryctl system login --provider None
+echo | mesheryctl mesh deploy adapter meshery-linkerd:10001 --token "./.github/workflows/auth.json"
 echo "Onboarding application... Standby for few minutes..."
-mesheryctl app onboard -f "./emojivoto.yml"
+mesheryctl app onboard -f "./emojivoto.yml" -s "Kubernetes Manifest" -t "./.github/workflows/auth.json"
```

Comment:
Let's just use `--token` here just for consistency

Reply:
```suggestion
mesheryctl app onboard -f "./emojivoto.yml" -s "Kubernetes Manifest" --token "./.github/workflows/auth.json"
```